source/_integrations/sun.markdown

```diff
@@ -38,23 +38,6 @@ elevation:
 <img src='/images/screenshots/more-info-dialog-sun.png' />
 </p>
 
-## Implementation Details
```

Comment:
Should we replace this with an actual example here instead?
I know there is an example on the automation page, but it might be fitting to have one here too?

Reply:
Would it not be better just to have the documentation in one place? I feel it might be better to reference the other documentation from here so it can be clicked through. The previous documentation really didn't make a lot of sense so I feel it's best just removed.

Reply:
> Would it not be better just to have the documentation in one place? 

Well yeah. The triggers page is to educate about triggers.
But if one looks for information about the sun integration, one would look on the integrations page. 

I don't see how adding an example to this page would bad?